### PR TITLE
Introduce flake8 for linting

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    funding_sources_module/*,
+    libs/*,
+    migrations/*,
+    models/*,
+    payouts_module/*,
+    scripts/*,
+    test_reports/*,
+    tests/*,
+    webhooks_module/*,
+    gunicorn_config.py
```
Comment (ashwin1111) [inline .flake8]: let's not exclude any of these for now

File: .flake8 [hunk 2]
```diff
@@ -0,0 +1,23 @@
+[flake8]
+extend-ignore = E712, E711
```
Comment (ashwin1111) [inline .flake8:2]: would be good to add a comment of what these are

File: .github/workflows/lint_and_format_check.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+name: Lint and Format Check
```
Comment (ashwin1111) [inline .github/workflows/lint_and_format_check.yml:1]: this is not needed since we run this on dockerfile

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+default_stages: [commit]
+fail_fast: true
+default_language_version:
+  python: python3
+exclude: 'venv|.venv|.git|README.md|beneficial_owners_module|customers_module|dev|dwolla_service|funding_sources_module|libs|migrations|models|payouts_module|scripts|test_reports|tests|webhooks_module|gunicorn_config.py'
```
Comment (ashwin1111) [inline .pre-commit-config.yaml]: remove all these

**Review by ashwin1111 — APPROVED** [review]

File: .flake8 [hunk 5]
```diff
@@ -3,7 +3,8 @@ extend-ignore =
     # Comparison to true should be 'if cond is true:' or 'if cond:'
     E712,
     # Comparison to None should be 'cond is None:' (E711)
-    E711
+    E711,
+    W503
```
Comment (ashwin1111) [inline .flake8:7]: add comments

**Review by ashwin1111 — APPROVED** [review]

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**